### PR TITLE
LBM1-2300 If we failed to fetch the rootfs once, cleanup local cache …

### PR DESCRIPTION
**Pull request by lital-lb**
…and retry

Due to the issue LBM1-2300 where inaguration failed due to local cache issues

### CODE REVIEW

File: inaugurator/ceremony.py [hunk 1]
```diff
@@ -303,7 +303,16 @@ def _loadKernelForKexecing(self, destination):
...
     def _doOsmosisFromSource(self, destination):
-        osmosiscleanup.OsmosisCleanup(destination, objectStorePath=self._localObjectStore)
+        cleanup = osmosiscleanup.OsmosisCleanup(destination, objectStorePath=self._localObjectStore)
+        try:
+            self._doOsmosisFromSourceUnsafe(destination)
+        except Exception as e:
+            logging.exception("Failed to osmosis from source")
+            cleanup.eraseEverything()
+            self._talkToServer.progress(dict(state='warning', message=str(e)))
+            self._doOsmosisFromSourceUnsafe(destination)
```
Comment (maor-lb) [inline inaugurator/ceremony.py]: @lital-lb @sashas-lb , is failing here generating any trace?

File: inaugurator/ceremony.py [hunk 2]
```diff
@@ -303,7 +303,16 @@ def _loadKernelForKexecing(self, destination):
             append=self._args.inauguratorPassthrough)
 
     def _doOsmosisFromSource(self, destination):
-        osmosiscleanup.OsmosisCleanup(destination, objectStorePath=self._localObjectStore)
+        cleanup = osmosiscleanup.OsmosisCleanup(destination, objectStorePath=self._localObjectStore)
+        try:
+            self._doOsmosisFromSourceUnsafe(destination)
+        except Exception as e:
+            logging.exception("Failed to osmosis from source")
+            cleanup.eraseEverything()
```
Comment (maor-lb) [inline inaugurator/ceremony.py]: @lital-lb why do we only delete the osmosis cache aren't we should delete everything include system dirs?

Comment (sashas-lb) [reply inaugurator/ceremony.py]: yep you are correct we probably also need to "clear" destination as well

File: inaugurator/ceremony.py (hunk 1 above)
Comment (lital-lb) [reply inaugurator/ceremony.py]: It should since there's no try/except

File: inaugurator/ceremony.py (hunk 2 above)
Comment (lital-lb) [reply inaugurator/ceremony.py]: I added the cleanup for destination in the next line

**Pull request merged**